v1/plugin/plugin_test.go

```diff
@@ -253,20 +279,15 @@ func TestMakeGRPCCredentials(t *testing.T) {
 				KeyPath:       tlsTestSrv + keyFileExt,
 				RootCertPaths: tlsTestCA + crtFileExt,
 			}
-			mockServerSetupInUse := newMockTLSSetup(tlsSetup)
-			tlsSetup = mockServerSetupInUse
 			var configReport *tls.Config
-			mockServerSetupInUse.doMakeTLSConfig = func() *tls.Config {
-				tlsConfig := mockServerSetupInUse.prevSetup.makeTLSConfig()
-				configReport = tlsConfig
-				return tlsConfig
-			}
+			mockServerSetupInUse := newMockTLSSetup(tlsSetup, &configReport, &m)
+			tlsSetup = mockServerSetupInUse
 			Convey("library should build GRPC credentials without issues", func() {
 				_, err := makeGRPCCredentials(&m)
 				So(err, ShouldBeNil)
 				Convey("certificate and client root certs should be loaded", func() {
 					So(configReport.Certificates, ShouldNotBeEmpty)
-					So(configReport.ClientCAs.Subjects(), ShouldNotBeEmpty)
+					So(configReport.RootCAs, ShouldNotBeNil) // ensure it's not nil
```

Comment:
Please remove this: 
// ensure it's not nil

Reply:
corrected